src/pad/README.md

```diff
@@ -1,21 +1,282 @@
+
+#### Examples
+```
+make_fake_io_site -name IO_HSITE -width 1 -height 204
+make_fake_io_site -name IO_VSITE -width 1 -height 200
+make_fake_io_site -name IO_CSITE -width 200 -height 204
+```
+
+### Remove IO Rows
+
+When the padring is complete, the following command can remove the IO rows to avoid causing confusion with the other tools
+```
+remove_io_rows
+```
+
+### Placing Pads
+
+To place a pad into the pad ring
+```
+place_pad -row row_name \
+          -location offset \
+          -mirror \
+          [-master master] \
+          name
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-row` | Name of the row to place the pad into, examples include: IO_NORTH, IO_SOUTH, IO_WEST, IO_EAST, IO_NORTH_0, IO_NORTH_1 |
+| `-location` | Offset from the bottom left chip edge to place the pad at. |
+| `-mirror` | Specifies if the pad should be mirrored. |
+| `-master` | Name of the instance master if the instance needs to be created. |
+| `name` | Name of the instance |
+
+#### Examples
+```
+place_pad -row IO_SOUTH -location 280.0 {u_clk.u_in}
+place_pad -row IO_SOUTH -location 360.0 -mirror {u_reset.u_in}
+place_pad -master sky130_fd_io__top_ground_hvc_wpad -row IO_SOUTH -location 439.5 {u_vzz_0}
+place_pad -master sky130_fd_io__top_power_hvc_wpad -row IO_SOUTH -location 517.5 {u_v18_0}
+```
+
+### Placing corners
+
+To place the corner cells
+```
+place_corners master \
+              [-ring_index index]
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-ring_index` | (Optional) Used to specify the index of the ring in case of multiple rings. |
+| `master` | Name of the master for the corners. |
+
+#### Examples
+```
+place_corners sky130_fd_io__corner_bus_overlay
+```
+
+### Placing IO Filler Cells
+
+To place the IO filler cells
+```
+place_io_fill -row row_name \
+              masters
+```
+
+#### Options
+
+| Switch Name | Description |
+| ----- | ----- |
+| `-row` | Name of the row to place the pad into, examples include: IO_NORTH, IO_SOUTH, IO_WEST, IO_EAST, IO_NORTH_0, IO_NORTH_1 |
+| `masters` | Names of the masters for the IO filler cells. |
+
+#### Examples
+```
+place_io_fill -row IO_NORTH s8iom0s8_com_bus_slice_10um s8iom0s8_com_bus_slice_5um s8iom0s8_com_bus_slice_1um
+place_io_fill -row IO_SOUTH s8iom0s8_com_bus_slice_10um s8iom0s8_com_bus_slice_5um s8iom0s8_com_bus_slice_1um
+place_io_fill -row IO_WEST s8iom0s8_com_bus_slice_10um s8iom0s8_com_bus_slice_5um s8iom0s8_com_bus_slice_1um
+place_io_fill -row IO_EAST s8iom0s8_com_bus_slice_10um s8iom0s8_com_bus_slice_5um s8iom0s8_com_bus_slice_1um
+```
+
+### Connecting Ring Signals
+
+Once the ring is complete, use the following command to connect the ring signals
+```
+connect_by_abutment
+```
+
+### Placing Wirebond Pads
+
+To place the wirebond pads over the IO cells
+```
+place_bondpad -bond master
+              [-offset {x y}] \
+              [-rotation rotation] \
+              io_instances
+```
```

Comment:
Wirebond pads cannot be placed before the io cells are placed. I assume you mean the `*bump_array` commands, which I decided to put below since not all designs will those and this keeps the cover cell handling together in the documentation